qemu/tests/fio_perf.py

```diff
@@ -209,7 +209,7 @@ def fio_install(tarball):
                 if s:
                     raise exceptions.TestFail("Failed to online disk: %s" % o)
     # format disk
-    if format:
+    if format == "True":
```

Comment:
I see. I tested in my case with "self.params.get("format")" and it returned a bool value. The statement in your case "params.get("format")" would return a str value. Your update is credible. Thanks.